### PR TITLE
Change min > max notation to [min, max] in Parameters Markdown

### PR DESCRIPTION
**Pull request by junwoo091400**

### CODE REVIEW

File: src/lib/parameters/px4params/markdownout.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def __init__(self, groups):
                         min_val='?'
                     if not max_val:
                         max_val='?'
-                    max_min_combined+='%s > %s ' % (min_val, max_val)
+                    max_min_combined+='%s, %s ' % (min_val, max_val)
```
Comment (tstastny) [inline src/lib/parameters/px4params/markdownout.py:71]:
```suggestion
                    max_min_combined+='[%s, %s] ' % (min_val, max_val)
```

**Review by tstastny — CHANGES_REQUESTED** [review]
What about keeping the brackets for the actual values?

File: src/lib/parameters/px4params/markdownout.py (hunk 1 above)
Comment (hamishwillee) [reply src/lib/parameters/px4params/markdownout.py:71]: I prefer this. Let's try it out. Can always revert.

**Review by hamishwillee — APPROVED** [review]
Tested. It is better. Thanks!

**Pull request merged**